src/aap_eda/tasks/ruleset.py

```diff
@@ -130,14 +131,14 @@ def enqueue_restart_task(
     ssl_verify: str,
 ) -> None:
     time_at = timezone.now() + timedelta(seconds=seconds)
-    get_scheduler().enqueue_at(
+    logger.info("Task started: enqueue_restart_task")
+    logger.info("Enqueueing restart task for activation id: %s", activation_id)
+    get_scheduler(name="activation").enqueue_at(
         time_at,
         activate_rulesets,
-        args=(
-            True,
-            activation_id,
-            deployment_type,
-            ws_base_url,
-            ssl_verify,
-        ),
+        True,
```

Comment:
@Alex-Izquierdo If this is new can we just use the activation_id alone since the rest of the parameters can be either fetched from the settings or are present in the activation instance.
* ws_base_url       from settings
* ssl_verify            from settings

Reply:
Well, this was a hotfix, but I have no problem to do such a refactor. Better in another PR?